GameServer/skillhandler/TripleWieldAbilityHandler.cs

```diff
@@ -23,7 +23,7 @@ public class TripleWieldAbilityHandler : IAbilityActionHandler
 		/// <summary>
 		/// The ability reuse time in seconds
 		/// </summary>
-		protected const int REUSE_TIMER = 7 * 60; // 7 minutes 
+		protected const int REUSE_TIMER = 10 * 60; // clait: 10 minutes [og: 7] 

```

Comment:
I don't know about the others but I just checked on live now and Triple Wield is still 7 minutes, so I wouldn't go about changing it here.